chartfx-chart/src/main/resources/de/gsi/chart/solid-pick.css

```diff
@@ -119,269 +119,202 @@ is being used to size a border should also be in pixels.
+  /* A little lighter than -fx-base and used as the -fx-color for the
      * "hovered" pseudoclass state.
      */
-    -fx-hover-base: ladder(
-        -fx-base,
-        derive(-fx-base,20%) 20%,
-        derive(-fx-base,30%) 35%,
-        derive(-fx-base,40%) 50%
-     );
-
-    /* A little darker than -fx-base and used as the -fx-color for the
+  -fx-hover-base: ladder(-fx-base, derive(-fx-base, 20%) 20%, derive(-fx-base, 30%) 35%, derive(-fx-base, 40%) 50%);
+
+  /* A little darker than -fx-base and used as the -fx-color for the
      * "armed" pseudoclass state.
      *
      * TODO: should this be renamed to -fx-armed-base?
      */
-    -fx-pressed-base: derive(-fx-base,-6%);
+  -fx-pressed-base: derive(-fx-base, -6%);
 
-    /* The color to use for -fx-text-fill when text is to be painted on top of
+  /* The color to use for -fx-text-fill when text is to be painted on top of
      * a background filled with the -fx-background color.
      */
-    -fx-text-background-color: ladder(
-        -fx-background,
-        -fx-light-text-color 45%,
-        -fx-dark-text-color  46%,
-        -fx-dark-text-color  59%,
-        -fx-mid-text-color   60%
-    );
-
-    /* A little darker than -fx-color and used to draw boxes around objects such
+  -fx-text-background-color: ladder(-fx-background, -fx-light-text-color 45%, -fx-dark-text-color 46%, -fx-dark-text-color 59%, -fx-mid-text-color
+    60%);
+
+  /* A little darker than -fx-color and used to draw boxes around objects such
      * as progress bars, scroll bars, scroll panes, trees, tables, and lists.
      */
-    -fx-box-border: ladder(
-        -fx-color,
-        black 20%,
-        derive(-fx-color,-15%) 30%
-    );
+  -fx-box-border: ladder(-fx-color, black 20%, derive(-fx-color, -15%) 30%);
 
-    /* Darker than -fx-background and used to draw boxes around text boxes and
+  /* Darker than -fx-background and used to draw boxes around text boxes and
      * password boxes.
      */
-    -fx-text-box-border: ladder(
-        -fx-background,
-        black 10%,
-        derive(-fx-background, -15%) 30%
-    );
+  -fx-text-box-border: ladder(-fx-background, black 10%, derive(-fx-background, -15%) 30%);
 
-    /* Lighter than -fx-background and used to provide a small highlight when
+  /* Lighter than -fx-background and used to provide a small highlight when
      * needed on top of -fx-background. This is never a shadow in Modena but
      * keep -fx-shadow-highlight-color name to be compatible with Caspian.
      */
-    -fx-shadow-highlight-color: ladder(
-        -fx-background,
-        rgba(255,255,255,0.07) 0%,
-        rgba(255,255,255,0.07) 20%,
-        rgba(255,255,255,0.07) 70%,
-        rgba(255,255,255,0.7) 90%,
-        rgba(255,255,255,0.75) 100%
-      );
-
-    /* A gradient that goes from a little darker than -fx-color on the top to
+  -fx-shadow-highlight-color: ladder(-fx-background, rgba(255, 255, 255, 0.07) 0%, rgba(255, 255, 255, 0.07) 20%,
```

Comment:
Codacy found an issue: [Expected newline after "(" in a multi-line function (function-parentheses-newline-inside)](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780860)